Snowflake.Data/Client/SnowflakeDbCommand.cs

```diff
@@ -22,13 +22,13 @@ public class SnowflakeDbCommand : DbCommand
 
         private SnowflakeDbParameterCollection parameterCollection;
 
-        private SFLogger logger = SFLoggerFactory.GetLogger<SnowflakeDbCommand>();
+        private SFLoggerPair _loggerPair = SFLoggerPair.GetLoggerPair<SnowflakeDbCommand>();

```

Comment:
would it be possible not to be forced to change this in all places in which we log anything?
Maybe SFLoggerPair could implement SFLogger interface?
Maybe SFLoggerFactory.GetLogger could return the same as SFLoggerPair.GetLoggerPair?
Maybe SFLoggerPair could do LogDebug -> Debug translations?
Maybe SFLoggerPair could mask secrets before passing them further?

Currently because of this 80 files are changed so it is even difficult to review.

Reply:
Made the following changes:
- SFLoggerPair implements SFLogger
- Replaced SFLoggerPair.GetLoggerPair with SFLoggerFactory.GetLogger
- Renamed LogDebug to Debug and other Log functions
- Masked secrets before passing to loggers